data/serializer/src/main/java/tech/pegasys/teku/provider/JsonProvider.java

```diff
@@ -45,6 +45,7 @@ private void addTekuMappers() {
     module.addSerializer(Bytes4.class, new Bytes4Serializer());
     module.addDeserializer(Bytes.class, new BytesDeserializer());
     module.addSerializer(Bytes.class, new BytesSerializer());
+    module.addSerializer(Double.class, new DoubleSerializer());
```

Comment:
Let's also add a deserializer and set up add some tests for de/serialization of doubles to `JsonProviderTest`

Reply:
Yes, indeed, sorry about missing it. Thank you!